api/hypertable/create_chunk.md

```diff
@@ -0,0 +1,92 @@
+---
+api_name: create_chunk()
+excerpt: Create a chunk with specified dimensional constraints
+topics: [hypertables]
+keywords: [chunks, hypertables, create]
+api:
+  license: apache
+  type: function
+products: [cloud, mst, self_hosted]
+---
+
+# create_chunk()
+
+Manually create a chunk with specific time ranges and space partition boundaries in a [$HYPERTABLE][hypertable-docs]. 
```

Comment:
We want to deemphasize space partitioning in docs so the most prominent examples should probably be time dimension only